runtime/doc/news.txt

```diff
@@ -393,6 +393,8 @@ The following changes to existing APIs or features add new behavior.
 • |:checkhealth| buffer can now be opened in a split window using modifiers like
   |:vertical|, |:horizontal| and |:botright|.
 
+• `--startuptime` now reports the startup times per process individually.
```

Comment:
```suggestion
• |--startuptime| now reports the startup times per process individually.
```

Reply:
just for my own understanding, `|foo|` is a link to the help, right? How can I know when I need to use `|foo|`?

Reply:
the help docs give lots of examples :) 

use `|tag|` whenever there is a `*tag*` defined somewhere.